### PR TITLE
feat: add ddl task service

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. Removes `Create` in the Router
2. Adds DDL task service

## Checklist

- [ ]  I have written the necessary comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: proto/greptime/v1/meta/ddl.proto [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    // The following information plays a bigger role in making messages traceable
+    // and facilitating debugging.
+    string subject = 2;
+    string from = 3;
+    string to = 4;
+    // The unix timestamp in milliseconds.
+    int64 timestamp_millis = 5;
+
+    bool cancelled = 6;
+}
```
Comment (fengjiachun) [inline proto/greptime/v1/meta/ddl.proto:85]: new line

File: proto/greptime/v1/meta/ddl.proto [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+package greptime.v1.meta;
+
+option go_package = "github.com/GreptimeTeam/greptime-proto/go/greptime/v1/meta";
+
+import "greptime/v1/meta/common.proto";
+
+service DdlTask {
+    // Submits a DDL task to meta.
+    rpc SubmitDdlTask(AddDdlTaskRequest) returns (AddDdlTaskResponse);
```
Comment (fengjiachun) [inline proto/greptime/v1/meta/ddl.proto:25]: How to ask for the execution status?

Comment (WenyXu) [reply proto/greptime/v1/meta/ddl.proto:25]: It returns directly in the response.

**Comment by WenyXu** [comment]
PTAL @MichaelScofield @fengjiachun

**Review by MichaelScofield — APPROVED** [review]

**Review by fengjiachun — APPROVED** [review]

**Pull request merged**